package.json

```diff
@@ -20,33 +21,50 @@
   },
   "homepage": "https://github.com/kids-first/kf-arranger#readme",
   "dependencies": {
-    "@kfarranger/server": "^1.2.2",
+    "@kfarranger/server": "^1.2.3",
```

Comment:
1.2.3 needed for this?

Reply:
I think so, though I can't remember why on top of my head.